content/applications/hr/lunch/user-accounts.rst

```diff
@@ -0,0 +1,106 @@
+     the account when paychecks are issued. This requires :ref:`adding a salary attachment
+     <payroll/worked-days-inputs>` for the user's payslip in the *Payroll* app.
+   - Companies can sell "lunch tickets" at a set price (for example, one ticket costs $5.00). Users
+     can purchase tickets from a *Lunch* app manager, who then updates the user's account.
+
+.. _lunch/cash-moves:
+
+Cash Moves
+==========
+
+To add funds to user accounts, each cash move must be individually logged. To view all cash move
+records, or create a new cash move, navigate to :menuselection:`Lunch app --> Manager --> Cash
+Moves`.
+
+All cash moves are presented in a list view, displaying each record's :guilabel:`Date`,
+:guilabel:`User`, :guilabel:`Description`, and :guilabel:`Amount` for each record. The total of all
+the cash moves is displayed at the bottom of the :guilabel:`Amount` column.
+
+.. image:: user-accounts/cash.png
+   :align: center
+   :alt: The list view of all cash moves.
+
+Add funds
+---------
+
+To add funds to a lunch account, click the :guilabel:`New` button, located in the top-left corner of
+the :guilabel:`Cash Moves` dashboard.
+
+A blank :guilabel:`Cash Moves` form loads. Enter the following information on the form:
+
+- :guilabel:`User`: select the user depositing cash into their account from the drop-down menu. If
+  the user is not in the database, they can be created by clicking either :guilabel:`Create "user"`
+  or :guilabel:`Create and edit...` to create the user and edit the :guilabel:`Create User` form.
+- :guilabel:`Date`: using the calendar popover, select the date the transaction occurred.
+- :guilabel:`Amount`: enter the amount being added to the lunch account.
+- :guilabel:`Description`: enter a brief description of the transaction.
+
+.. image:: user-accounts/cash-move-form.png
+   :align: center
+   :alt: The cash move form filled out for a transaction of $40.00.
+
+The cash move form automatically saves as data is entered, but the form can be saved manually at any
+time by clicking the *save manually* option, represented by a :guilabel:`(cloud upload)` icon,
+located in the top-left corner of the page. To delete the record and return to the *Cash Moves*
+dashboard, click the :guilabel:`↺ (Discard changes)` icon.
+
+Control Accounts
+================
+
+An overview of every transaction in the *Lunch* app, including all cash deposits and purchases, can
+be viewed on the main *Control Accounts* dashboard. To access this, navigate to
+:menuselection:`Lunch app --> Manager --> Control Accounts.`
+
+All transactions are grouped :guilabel:`By Employee`, and listed alphabetically by the user's first
+name. At the end of the user's name, a number appears. This indicates the number of individual
+records logged for that user.
+
+The default view is to have all individual transactions hidden. To view all transactions for a user,
+click the :guilabel:`▶ (triangle)` icon at the end of the name to expand the group.
```

Comment:
```suggestion
click the :guilabel:`▶ (triangle)` icon to the left of the desired name to expand that specific group.
```

Reply:
Can't you also reveal the tucked list by clicking on the user name? If so, you may want to consider adding that in...just a suggestion